### PR TITLE
Update UserGuide.md

### PR DESCRIPTION
**Pull request by isaacsaw25**
Add example outputs and append missing `generate` command description

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -16,8 +16,12 @@ src/main/resources/docs/
 *.iml
 bin/
 
+# IO output files
```
Comment (ehz0ah) [inline .gitignore:19]: Are the files remove from cache already? The IO test for GitHub Actions depends on it

**Review by ehz0ah — APPROVED** [review]
ACK

File: docs/UserGuide.md [hunk 2]
```diff
@@ -232,15 +284,40 @@ Example output:
...
+Format: `generate`
+Example output:
+```shell
+Here is the allocation report:
+┌───────────┬───────────────────────────────┐
+│ Student   │ University Granted            │
+├───────────┼───────────────────────────────┤
+│ A1234567I │ Chongqing University          │
+│ A2113113X │ National Tsing Hua University │
+└───────────┴───────────────────────────────┘
```
Comment (Holy-An) [inline docs/UserGuide.md:319]: Looks neat and nice

File: docs/UserGuide.md [hunk 3]
```diff
@@ -117,14 +168,16 @@ e.g.
...
-Shows a message explaining how to use the program. (Commands, etc.)  
-Format: `help`
+Deletes the student with the specified id.
+The id to be deleted must match the id in the student records.  
+e.g. `delete A1234567I`  
+Example output: 
+```shell
+> delete A1234567I
+--------------------------------------------------------------------------------
+Removed student, 1 student(s) left
```
Comment (Holy-An) [inline docs/UserGuide.md:178]: woah

**Review by Holy-An — APPROVED** [review]
LGTM

**Pull request merged**